### PR TITLE
Change migrated-to annoation key to follow best practices by removing beta

### PR DESCRIPTION
**Pull request by davidz627**
This has not made it into any official release and just sat in master for a couple days before we caught it. I think it's safe to change.

/kind api-change
/kind cleanup
/assign @msau42 @liggitt @saad-ali 

```release-note
CSI Migration migration annotation key is now `kubernetes.io/migrated-to`
```

### CODE REVIEW

File: pkg/controller/volume/persistentvolume/util/util.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ const (
 	// through the CSIMigration feature flags. When this annotation is set the
 	// Kubernetes components will "stand-down" and the external-provisioner will
 	// act on the objects
-	AnnMigratedTo = "volume.beta.kubernetes.io/migrated-to"
+	AnnMigratedTo = "kubernetes.io/migrated-to"
```
Comment (msau42) [inline pkg/controller/volume/persistentvolume/util/util.go]: I think we still want a "volume" prefix

Comment (davidz627) [reply pkg/controller/volume/persistentvolume/util/util.go]: based on the other keys it looks like `pv` might actually be the appropriate prefix since it is applied to both `pv/pvc`. WDYT?

Comment (davidz627) [reply pkg/controller/volume/persistentvolume/util/util.go]: looking at `AnnBoundByController` as a template

Comment (msau42) [reply pkg/controller/volume/persistentvolume/util/util.go]: We used `pv` slightly more than `volume` so that seems fine. I don't have a strong preference either way.

File: go.sum [hunk 2]
```diff
@@ -621,6 +621,8 @@ mvdan.cc/unparam v0.0.0-20190209190245-fbb59629db34/go.mod h1:H6SUd1XjIs+qQCyskX
 rsc.io/pdf v0.1.1/go.mod h1:n8OzWcQ6Sp37PL01nO98y4iUCRdTGarVfzxY20ICaU4=
 sigs.k8s.io/kustomize v2.0.3+incompatible h1:JUufWFNlI44MdtnjUqVnvh29rR37PQFzPbLXqhyOyX0=
 sigs.k8s.io/kustomize v2.0.3+incompatible/go.mod h1:MkjgH3RdOWrievjo6c9T245dYlB5QeXV4WCbnt/PEpU=
+sigs.k8s.io/structured-merge-diff v0.0.0-20190525122527-15d366b2352e h1:4Z09Hglb792X0kfOBBJUPFEyvVfQWrYT/l8h5EKA6JQ=
+sigs.k8s.io/structured-merge-diff v0.0.0-20190525122527-15d366b2352e/go.mod h1:wWxsB5ozmmv/SG7nM11ayaAW51xMvak/t1r0CSlcokI=
```
Comment (liggitt) [inline go.sum]: all the go.sum changes look unrelated... drop them?
